ops/terraform/services/pipeline/user-data.sh.tftpl

```diff
@@ -28,7 +35,7 @@ aws ssm get-parameters-by-path \
 cat <<EOF > extra_vars.json
 {
   "data_pipeline_db_url": "${writer_endpoint}",
-  "data_pipeline_jvm_args": "-Xmx{{ ansible_memtotal_mb * 0.80 | int }}m -XX:+PreserveFramePointer",
+  "data_pipeline_jvm_args": "-Xmx{{ ((ansible_memtotal_mb * 0.80) | int) }}m -XX:+PreserveFramePointer",
   "data_pipeline_loader_threads": "{{ ansible_processor_vcpus * 25 }}",
   "data_pipeline_new_relic_app_name": "BFD Pipeline ({{ env_name_std }})",
```

Comment:
I spoke with @brianburton about this, and we didn't necessarily come to any conclusions, but leaving it as it is **will** make some of the data in the PACA dashboards for these RDA jobs less meaningful. As far as I understand it, _environment_ or similar isn't a first-class facet in New Relic, and differentiation of metrics is done through fully different application names. We discussed accepting a "breaking change" for their data visualizations by doing this or similar:
```suggestion
  "data_pipeline_new_relic_app_name": "BFD Pipeline ${pipeline_instance} ({{ env_name_std }})",
```